src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -842,7 +842,7 @@ private async ValueTask ReadHeadersAsync(long headersLength, CancellationToken c
             // https://tools.ietf.org/html/draft-ietf-quic-http-24#section-4.1.1
             if (headersLength > _headerBudgetRemaining)
             {
-                _stream.Abort(QuicAbortDirection.Write, (long)Http3ErrorCode.ExcessiveLoad);
+                _stream.Abort(QuicAbortDirection.Read, (long)Http3ErrorCode.ExcessiveLoad);
```

Comment:
With the previous behavior of closing the write direction, were things still eventually being cleaned up correctly?

Reply:
Yes, you can sort of see it ion the posted logs. Frirst comes from the ReadHeadersAsync above. 

```
        01:40:10.3532604[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 263.

        01:40:10.3546351[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Write with 258.
        01:40:10.3547886[Info] thisOrContextObject: QuicStream#41727345, memberName: Abort, message: [strm][0x7FA554013E40] Aborting Read with 268.
```
I think this comes from HTTP3 error processing. This is easy to reproduce so I can look where this duplicate reset is coming from.